apps/vanilla/src/pages/home/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
+    <link rel="stylesheet" href="../index.css" />
+    <title>Vite App</title>
+  </head>
+  <header class="header">
+
+  </header>
+  <body class="default-layout">
+    <div class="wrapper">
+      <h1 class="title">Saritasa Anime</h1>
+      <div class="search">
+        <h1 class="search__title">Search</h1>
+        <div class="search__form">
+          <input class="search__form-input" />
+          <button class="btn search__form-button" type="button">Search</button>
+        </div>
+      </div>
+       <!-- Filter -->
+       <div class="filter">
+        <div class="filter__item">
+          <label class="filter__item_title">Sort by</label>
+          <select class="filter__item_select filter__item_select-sort"></select>
+        </div>
```

Comment:
I think you're a bit confused with BEM. In this case, `div`, `label` and `select` have different purposes, so they can't be same element. But `select`s essentially do the same thing, but differ in options, so should use modifier.
```suggestion
        <div class="filter__item">
          <label class="filter__title">Sort by</label>
          <select class="filter__select filter__select_sort"></select>
        </div>
```

Reply:
Also please fix indentation, you can use prettier or vscode formatter.